### ISSUE TITLE
Update DS Web Acceptance User Group

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#1815]
Right notification for all DS folder updates are automatically sent to ABC Trans list, but they should be sent to ROW Intake Group. 

Since these updates need to be manually identified and updated, it's easy for them to get overlooked for a while. We don't know who oversees the ABC Trans list, but could be annoying for them, too. 

**The need to address this bug will increase if applications move fully online.**  

*Migrated from [atd-amanda #202](https://github.com/cityofaustin/atd-amanda/issues/202)*

**Comment by amenity** [comment]
Confirmed that this is still happening on 3/17/20. :)

**Comment by amenity** [comment]
Right now DS folder is automatically sent to ABC Trans list instead of ROW Intake Group, ROW has to manually correct. 

Not sure if it's annoying to ABC Trans list, but for ROW this increases the chances that they will miss the request. If all applicants are submitting through portal, things are more likely to fall through the cracks.

**Comment by SurbhiBakshi** [comment]
meeting Paloma to get more details - 3/23

**Comment by SurbhiBakshi** [comment]
![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/df3c4ae9-db92-48cc-9c52-4d33f2371280)

**Comment by SurbhiBakshi** [comment]
For DS permits, change default assigned user from ABC Trans to ROW Intake Group.

Folder Type - DS
Status - Internet Pending
Under Process tab, Web Application Acceptance
Assigned User should default to ROW Intake Group and not ABC Trans

**Comment by SurbhiBakshi** [comment]
![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/a52cf13a-554c-497a-8b88-4e0817238837)

**Comment by SurbhiBakshi** [comment]
sent in a ticket

**Comment by SurbhiBakshi** [comment]
Have asked for an update on this CTM ticket - what is the current status and who this ticket has been assigned to?

**Comment by SurbhiBakshi** [comment]
Good morning,
 
Email from CTM ref. to update request - Thank you for contacting the Service Desk about your request (REQ0042141). I understand you are seeking an update, as this request was submitted some time ago. I apologize for the delay in our response and will be happy to assist you further.
After some searching, the ticket appears to have been assigned to AMANDA_SUPPORT_SERVICES group for processing. In this case, I can reach out to a member of this group for more information. A member of IT support staff should be reaching out to you with additional steps soon. That said, please don't hesitate to let us know if you have any other questions or concerns in the meantime.
 
Thank you
 
Bahoua D

**Comment by SurbhiBakshi** [comment]
Swapna has incorporate the fix in Dev and Test. Waiting for the fix to be tested before this gets moved to Prod.

**Comment by Nadin-Nader** [comment]
@SurbhiBakshi I was able to test this on TEST with and without Swapna on Friday. I was able to create a DS folder  (FolderRSN 12387164) and it Assigned it to the ROWINTAKE Group on the PROCESS tab. The fix is ready for PROD.

![image](https://user-images.githubusercontent.com/63309637/79130044-f1cdf200-7d6b-11ea-815e-43beec7e683a.png)

![image](https://user-images.githubusercontent.com/63309637/79130084-014d3b00-7d6c-11ea-9387-0014b232995a.png)
